x-pack/plugins/secops/server/graphql/ecs/resolvers.test.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { toStringArrayScalar } from './resolvers';
+
+describe('Test ToStringArray Scalar Resolver', () => {
+  describe('#serialize', () => {
+    test('Test Null String', () => {
+      expect(toStringArrayScalar.serialize(null)).toEqual(null);
+    });
+
+    test('Test Undefined String', () => {
+      expect(toStringArrayScalar.serialize(undefined)).toEqual(null);
+    });
+
+    test('Test Basic String', () => {
+      expect(toStringArrayScalar.serialize('hello')).toEqual(['hello']);
+    });
+
+    test('Test Basic Number', () => {
+      expect(toStringArrayScalar.serialize(33)).toEqual(['33']);
+    });
+
```

Comment:
Here are two more unit tests I used during this PR review to include which both pass so many thanks for the unit tests! 👍 

These two tests, use a circular reference to ensure that `Invalid Object` is returned:

```suggestion
    test('Test Simple Circular Reference', () => {
      const circularReference = { myself: {} };
      circularReference.myself = circularReference;
      expect(toStringArrayScalar.serialize(circularReference)).toEqual(['Invalid Object']);
    });

    test('Test Simple Complex Circular Reference', () => {
      const circularReference = { myself: {} };
      circularReference.myself = circularReference;
      expect(toStringArrayScalar.serialize([10, 20, 30, 40, circularReference])).toEqual([
        '10',
        '20',
        '30',
        '40',
        'Invalid Object',
      ]);
    });
```

Reply:
Thanks